scripts.js

```diff
@@ -1,19 +1,26 @@
 function add_doggo() {
-    fetch("https://dog.ceo/api/breed/shiba/images")
+    fetch("https://dog.ceo/api/breed/shiba/images/random")
         .then(function(response) {
             return response.json();
         })
         .then(function(imageObj) {
-            show_image(imageObj.message[0], 500,500, "Doggo");
+            show_image(imageObj.message, 500,500, "Doggo");
```

Comment:
if width, height, and alt are always going to be the same, you can also just have those in the function instead to make this even leaner....
```javascript
fetch("https://dog.ceo/api/breed/shiba/images/random")
  .then(response => response.json())
  .then(show_image);
```

Reply:
fixed